plugins/source/gcp/go.mod

```diff
@@ -68,12 +70,20 @@ require (
 // TODO: remove once all updates are merged
 replace github.com/apache/arrow/go/v14 => github.com/cloudquery/arrow/go/v14 v14.0.0-20230916001126-ffb7089245ac
 
+// https://github.com/invopop/jsonschema/pull/103
+replace github.com/invopop/jsonschema => github.com/cloudquery/jsonschema v0.0.0-20230926171440-9c0d982c4f4f
+
+// https://github.com/cloudquery/codegen/pull/29
+replace github.com/cloudquery/codegen => github.com/cloudquery/codegen v0.3.3-0.20230928065241-2c7e0070ce1e
```

Comment:
waiting for https://github.com/invopop/jsonschema/pull/103 & https://github.com/cloudquery/codegen/pull/29